addons/website/data/website_data.xml

```diff
@@ -115,6 +115,137 @@
         </record>
     </data>
 
+    <template id="cookie_policy" name="Cookie Policy">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="oe_structure">
+                    <section class="pt8 pb8">
+                        <div class="container">
+                            <h2 class="pt16">Cookies</h2>
+                            <p>
+                                Cookies are small bits of text sent by our servers to your computer or device when you access our services.
+                                They are stored in your browser and later sent back to our servers so that we can provide contextual content.
+                                Without cookies, using the web would be a much more frustrating experience.
+                                We use them to support your activities on our website, for example your session (so you don't have to login again) or your shopping cart.
```

Comment:
We use them to support your activities on our website. For example, your session (so you don't have to login again) or your shopping cart.